src/quadrature/Sn_Ordinate_Space.cc

```diff
@@ -45,7 +44,7 @@ vector<Moment> Sn_Ordinate_Space::compute_n2lk_1Da_(Quadrature_Class, unsigned /
   for (int ell = 0; ell <= static_cast<int>(L); ++ell)
     for (int k = 0; k <= ell; ++k)
       if ((ell + k) % 2 == 0)
-        result.push_back(Moment(ell, k));
+        result.emplace_back(Moment(ell, k));
```

Comment:
Should we always use `emplace_back` (or `emplace` more maps) instead of `push_back` when augmenting vectors (or maps)?

Reply:
In most cases `emplace_back` will be the correct choice.  While, I didn't try it here, we might be able to simply call
```c++
result.emplace_back(ell,k);
```
since the construction call is implied.

Reply:
* Ref: https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-emplace.html
* `emplace_back` usually results in less verbose code.  It often generates more efficient assembly.

Reply:
Great - that's good to know!